doc/classes/PhysicsServer3D.xml

```diff
@@ -1014,10 +1014,262 @@
+			<description>
+				Sets the drag coefficient of a soft body. Higher values increase this body's air resistance.
+				[b]Note:[/b] This value is currently unused by Godot's default physics implementation.
+			</description>
+		</method>
+		<method name="soft_body_set_linear_stiffness">
+			<return type="void" />
+			<param index="0" name="body" type="RID" />
+			<param index="1" name="stiffness" type="float" />
+			<description>
+				Sets the linear stiffness of a soft body. Higher values will result in a stiffer body, while lower values will increase the body's ability to bend. The value can be between [code]0.0[/code] and [code]1.0[/code] (inclusive).
+			</description>
+		</method>
+		<method name="soft_body_set_mesh">
+			<return type="void" />
+			<param index="0" name="body" type="RID" />
+			<param index="1" name="mesh" type="RID" />
+			<description>
+				Sets the mesh of a soft body.
+				[b]Note:[/b] The tessellation of the mesh matters, as the vertices and edges of the mesh faces will become the nodes and constraints of the soft body.
+			</description>
+		</method>
+		<method name="soft_body_set_pressure_coefficient">
+			<return type="void" />
+			<param index="0" name="body" type="RID" />
+			<param index="1" name="pressure_coefficient" type="float" />
+			<description>
+				Sets the pressure coefficient of a soft body. Simulates pressure build-up from inside this body. Higher values increase the strength of this effect.
+			</description>
+		</method>
+		<method name="soft_body_set_ray_pickable">
+			<return type="void" />
+			<param index="0" name="body" type="RID" />
+			<param index="1" name="enable" type="bool" />
+			<description>
+				Sets whether a soft body will be pickable when using object picking.
+			</description>
+		</method>
+		<method name="soft_body_set_simulation_precision">
+			<return type="void" />
+			<param index="0" name="body" type="RID" />
+			<param index="1" name="simulation_precision" type="int" />
+			<description>
+				Sets the simulation precision of a soft body. Increasing this value will improve the resulting simulation, but can affect performance. Use with care.
+			</description>
+		</method>
+		<method name="soft_body_set_space">
+			<return type="void" />
+			<param index="0" name="body" type="RID" />
+			<param index="1" name="space" type="RID" />
+			<description>
+				Assigns a space to a soft body (see [method space_create]).
+			</description>
+		</method>
+		<method name="soft_body_set_state">
+			<return type="void" />
+			<param index="0" name="body" type="RID" />
+			<param index="1" name="state" type="int" enum="PhysicsServer3D.BodyState" />
+			<param index="2" name="variant" type="Variant" />
+			<description>
+				Sets a body state (see [enum BodyState] constants).
+				[b]Note:[/b] Godot Physics does not support [constant BODY_STATE_LINEAR_VELOCITY], [constant BODY_STATE_ANGULAR_VELOCITY], [constant BODY_STATE_SLEEPING] or [constant BODY_STATE_CAN_SLEEP].
```

Comment:
```suggestion
				[b]Note:[/b] Godot Physics does not support [constant BODY_STATE_LINEAR_VELOCITY], [constant BODY_STATE_ANGULAR_VELOCITY], [constant BODY_STATE_SLEEPING], or [constant BODY_STATE_CAN_SLEEP].
```
Please use [Oxford comma](https://docs.godotengine.org/en/latest/contributing/documentation/docs_writing_guidelines.html#use-the-oxford-comma-to-enumerate-anything).